docs/devctr-image.md

```diff
@@ -46,9 +46,14 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   Run the following command.
+
    ```bash
-    docker build -t fcuvm -f tools/devctr/Dockerfile.x86_64 .
-    ```
+    tools/devtool build_devctr
+   ```
+
+   This internally, also looks for outdated packages and tries to
```

Comment:
This phrase is not very clear on what happens during the update phase. It also does not specify the fact that we can disable updating.

Reply:
made some edits.